Mathlib/Data/Rat/Cast/Order.lean

```diff
@@ -263,11 +263,15 @@ def evalRatCast : PositivityExt where eval {u α} _zα _pα e := do
   let ~q(@Rat.cast _ (_) ($a : ℚ)) := e | throwError "not Rat.cast"
   match ← core q(inferInstance) q(inferInstance) a with
   | .positive pa =>
-    let _oα ← synthInstanceQ q(LinearOrderedField $α)
+    let _oα ← synthInstanceQ q(Field $α)
+    let _oα ← synthInstanceQ q(LinearOrder $α)
+    let _oα ← synthInstanceQ q(IsStrictOrderedRing $α)
```

Comment:
(again is it fine that all these instances have the same name? I don't know enough about metaprogramming to know if this is an issue)

Reply:
Note to self `let __ ← synthInstanceQ` works.